Publish-gh-pages script fails -- error: git clone failed

## 🐛 Bug Report

When trying to publish changes to our github pages hosted site, the npm run publish-gh-pages step fails with the following error code (full output from powershell attached)

generate.js triggered...
feed.js triggered...
feed.js triggered...
sitemap.js triggered...
Site built successfully. Generated files in 'build' folder.
7ce717117293691c87678f7b3cc7fbe015e946de
fatal: Too many arguments.
Error: git clone failed
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ publish-gh-pages: `docusaurus-publish`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ publish-gh-pages script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\artus\AppData\Roaming\npm-cache\_logs\2020-01-30T23_49_07_563Z-debug.log

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

## To Reproduce
Windows 10
node (12.14.1)
npm (6.13.4)
yarn (1.21.1)

1. clone documentation repo locally
2. make edits in visual studio code
3. run command: cmd /C "set GIT_USER=[myuser] && set CURRENT_BRANCH=master && npm run publish-gh-pages"
4. fails :(

Note -- also fails in the same way when using yarn run publish-gh-pages


## Expected behavior

My 1st time trying to push changes to our docusaurus site. I expected it to work without any errors. 

## Actual Behavior

npm run publish-gh-pages failed

## Reproducible Demo

Repo that I'm trying to push changes for
https://github.com/OpenBCI/Documentation/tree/master

[docusaurus-errortxt.txt](https://github.com/facebook/docusaurus/files/4136846/docusaurus-errortxt.txt)

--- a/docs/getting-started-publishing.md
+++ b/docs/getting-started-publishing.md
@@ -110,7 +110,7 @@ GIT_USER=<GIT_USER> \
 **Windows**
 
 ```batch
-cmd /C "set "GIT_USER=<GIT_USER>" && set CURRENT_BRANCH=master && set USE_SSH=true && yarn run publish-gh-pages"
+cmd /C "set "GIT_USER=<GIT_USER>"&& set CURRENT_BRANCH=master && set USE_SSH=true && yarn run publish-gh-pages"
 ```
 
 There are also two optional parameters that are set as environment variables:

